Source/Core/VideoCommon/Present.cpp

```diff
@@ -244,7 +245,11 @@ Presenter::ConvertStereoRectangle(const MathUtil::Rectangle<int>& rc) const
 
 float Presenter::CalculateDrawAspectRatio() const
 {
-  const auto aspect_mode = g_ActiveConfig.aspect_mode;
+  auto aspect_mode = g_ActiveConfig.aspect_mode;
+  if (IsVREnabled())
+  {
+    aspect_mode = AspectMode::AnalogWide;
```

Comment:
Why?

Reply:
In my VR integration there is currently no support for other aspect ratios. The aspect ratios has to be hardcoded for 3DoF/6DoF VR anyway.